cypress/e2e/activityHistoryProperty.feature

```diff
@@ -0,0 +1,32 @@
+@activity-history
+@authentication
+@common
+@processes
+@root
+
+Feature: Activity History for Property
+  I want to view a property's activity history
+
+  Background:
+    Given I am logged in
+
+  @SmokeTest
+  Scenario Outline: View property activity history page
+    Given I seeded the database with an asset
+    Given I go to the property activity history
+    Then the property activity history is displayed
+    And table headers should be visible
+    When I click close activity history
+    Then I am on the property page for the property
+
+
+# //TODO commented for 5th July release as this test is failing in pipeline
```

Comment:
Do we need this?

Reply:
Oops, no we don't

Reply:
c6387940b979ee04b174740c803a91ef9098c881